apps/web/src/components/Basenames/RegistrationForm/index.tsx

```diff
@@ -1,4 +1,3 @@
-import { ExclamationCircleIcon, InformationCircleIcon } from '@heroicons/react/16/solid';
```

Comment:
Would recommend enabling "Hide whitespace" before reviewing this file. The diff is much smaller than it seems